Components/Components.md

```diff
@@ -1,141 +1,313 @@
 # React Components Explained
 
-
 ## Table of Contents
-- [React Component ](#component-reconciliation)
+- [React Components](#what-are-components)
 - [Component Reconciliation](#component-reconciliation)
 - [JSX - Syntactic Sugar](#jsx---syntactic-sugar)
 - [Hooks: Bringing State and Lifecycle to Functional Components](#hooks-bringing-state-and-lifecycle-to-functional-components)
 - [Component Lifecycle](#component-lifecycle)
 - [Render Props and Higher-Order Components (HOCs)](#render-props-and-higher-order-components-hocs)
 - [Context API for Prop Drilling](#context-api-for-prop-drilling)
-- [Concurrent Mode and Suspense](#concurrent-mode-and-suspense)
```

Comment:
Remove this content because it is the higher hooks covered in the hooks topics seperately